### PR TITLE
Hotels map: create "drop" marker component

### LINKED ISSUE #58: "Drop" map marker
Please create map marker with drop shape. One picture instead of long description:

screen shot 2017-12-13 at 20 24 33

I am interested in your creative solution. Possible inspiration: http://browniefed.com/blog/the-shapes-of-react-native/ 🙂 

Zeplin: https://zpl.io/bo0Gqpg

### PR DESCRIPTION
**Pull request by mrtnzlml**
Note: I didn't do the white circle in the middle of marker and I hope
no-one will notice... :)) (I don't know how to do it easily)

Closes #58

Checklist:

- [x] it works in landscape and portrait mode
- [x] it works offline (if possible) otherwise `cacheConfig.force=true` is used

### CODE REVIEW

**Comment by mrtnzlml** [comment]
Please test these changes in [Expo](https://docs.expo.io/versions/latest/introduction/installation.html#mobile-client-expo-for-ios-and-android) after build has finished:

  QR Code

File: app/common/src/map/DropMarker.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ *   v
+ *   .-------.
+ *   |       |
+ *   |   x   |
+ *   |       |
+ *   `-------`
+ */
+export default function DropMarker({ size = 50 }: Props) {
+  return (
+    <View style={{ position: 'relative' }}>
```
Comment (jaroslav-kubicek) [inline app/common/src/map/DropMarker.js]: isn't `position: 'relative'` set by default in RN?

Comment (mrtnzlml) [reply app/common/src/map/DropMarker.js]: Yes, yes it is. I did it for a reason that is funny and it will never happen so I will rather remove it...

Comment (mrtnzlml) [reply app/common/src/map/DropMarker.js]: Ok, should be fixed now.

**Pull request merged**